### ISSUE TITLE
React newbie question

### ISSUE THREAD

**Issue by ken-murray** [vasturiano/react-force-graph#256]
I'm a newbie to React.  I want to include your graph as a class Component within App.js.  For context, App.js has a number of Tabs, each of which includes a component.  One of these tabs would show the graph.

I copied the code from the https://codesandbox.io/s/uqel4 example and it shows nothing, just a blank section.  Data is loaded and no associated console errors.  Maybe it's just a CSS issue.

Is it possible to provide a React example where the data and graph gets loaded within a component class included from App.js?  Spent hours looking on google :-(

**Comment by vasturiano** [comment]
@ken-murray the difference between a class type of component and a hooks based one (like those shown in the examples) is just syntax. You should be able to convert from one to the other form interchangeably.

It's also more of a React kind of question. If you are having any issues specific to this module, could you please create your own reduced example that reproduces the issue on codesandbox.io, so we can have a closer look.

**Comment by ken-murray** [comment]
Many thanks, I created a reduced/simplified project as you suggested and eventually resolved the numerous issues I had with my code.  Just one example: I had named the variable for the data in the render method "graphData" which just happens to be used by one of the JS library files so conflicted.  All resolved and working great now :-)

I have a lot of improvements to make with how I display the data, but now it's working I can learn more about your code.  Would be cool to have the data model slowly spinning after loading, but I'm sure I can find a way to do that.

Many thanks for writing this, it's brilliant.
